python/ray/data/aggregate.py

```diff
@@ -103,6 +105,60 @@ def __init__(self, on: AggregateOnT = None):
             name=(f"mean({str(on)})"))
 
 
+class Std(AggregateFn):
+    """Defines standard deviation aggregation."""
+
+    # Uses Welford's online method for an accumulator-style computation of the
```

Comment:
Side note RLlib also has one of these but generalized for multi-shape mean/std; but that's probably overkill for now.

Reply:
Ooh neat, I'll check that out

Reply:
Do you have a link?

Reply:
MeanStdFilter